Tasks/AzureKeyVaultV1/operations/azure-arm-keyvault.ts

```diff
@@ -5,6 +5,9 @@ import util = require("util");
 import tl = require('azure-pipelines-task-lib/task');
 import webClient = require("azure-pipelines-tasks-azure-arm-rest/webClient");
 
+let retriesCount: number;
```

Comment:
Let's avoid declaring global variables - declare those as a part of method signature

Reply:
modified the declaration as a part of method signature.